src/views/List.jsx

```diff
@@ -22,18 +23,31 @@ export function List({ data, listToken }) {
 		return <Navigate to="/" />;
 	}
 
+	//  const additemHandler = () => {
+	// 	return < Navigate to = '/add-item' />
+	//  }
+
 	return (
 		<>
-			<form>
-				<label htmlFor="search-input"> Filter items </label>
-				<input
-					type="search"
-					placeholder="Start typing here..."
-					name="search-item"
-					id="search-input"
-					onChange={handleChange}
-				/>
-			</form>
+			{data.length > 0 ? (
```

Comment:
checking if the data exist first before accessing its length will avoid some possible errors in the future. 
You can either use the "and operator" `data && data.length > 0`, or a shorter version with the optional chaining operator  `data?.length  > 0`

Reply:
yes, we worried about this while writing the code. Thanks for showing us a cleaner way of doing this. Suggestion implemented.